lib/plugins/servercookie.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Treasure Server Side Cookie
+ */
+
+var jsonp = require('jsonp')
+var noop = require('../utils/misc').noop
+var invariant = require('../utils/misc').invariant
+var cookie = require('../vendor/js-cookies')
+
+var cookieName = 'td_ssc_cookie'
+
+function cacheSuccess (result, cookieName) {
+  cookie.setItem(cookieName, result['td_ssc_id'], 6000)
+  return result['td_ssc_id']
+}
+
+function configure (config) {
+  if (config.useServerSideCookie) {
+    cookie.getItem(cookieName)
+  }
+  return this
+}
+
+function fetchServerCookie (success, error, forceFetch) {
+  success = success || noop
+  error = error || noop
+
+  var url = 'https://' + this.client.cookieDomainHost + '/get_cookie_id'
```

Comment:
@MikeMcElroy should this actually build the url using cookieDomain name?

Reply:
We talked about this in Slack, and cookie_domain should be collected to send over the wire to the API.

Reply:
Also, don't forget the cachebusting random value.